components/automate-ui/src/app/pages/+compliance/shared/reporting/stats.service.ts

```diff
@@ -65,22 +66,57 @@ export class StatsService {
     return this.httpClient.post<any>(url, body).pipe(
       map(({ controls_summary }) => controls_summary));
   }
-
+  gettimeTrends(days: moment.DurationInputArg1 , intervals: moment.unitOfTime.DurationConstructor , reportQuery: ReportQuery){
+    reportQuery.startDate = moment.utc(reportQuery.endDate).subtract(days , intervals)
+  }
+  checktime(time_interval: number , reportQuery : ReportQuery) {
```

Comment:
@VivekSharma2607 please write the correct syntax 
`time_interval: number , reportQuery : ReportQuery is wrong `
`time_interval: number , reportQuery: ReportQuery`

Reply:
also check all the code changes and indentation.